pkg/etcd/etcd.go

```diff
@@ -329,6 +329,16 @@ func (e *ETCD) join(ctx context.Context, clientAccessInfo *clientaccess.Info) er
 	}
 
 	for _, member := range members.Members {
+		lastHyphen := strings.LastIndex(member.Name, "-")
+		memberNodeName := member.Name[:lastHyphen]
+		if memberNodeName == e.config.ServerNodeName {
```

Comment:
We're checking to see if the member name (minus the -uuid suffix) matches ServerNodeName, but we use `strings.SplitN(e.config.ServerNodeName, ".", 2)` when setting the member name - shouldn't we be consistent in both setting and checking either the short host name, or the FQDN?